src/codeu/chat/client/commandline/Panel.java

```diff
@@ -30,7 +33,46 @@
     void invoke(List<String> args);
   }
 
+  class TimerCommand {
```

Comment:
Sorry, I'm confused. What do you mean by Action? Do you mean the Action class in the javax.swing package?